src/settings/StaffSlips/StaffSlipForm.test.js

```diff
@@ -0,0 +1,201 @@
+import React from 'react';
+import {
+  render,
+  screen,
+  fireEvent,
+} from '@testing-library/react';
+
+import '../../../test/jest/__mock__';
+
+import { Field } from 'react-final-form';
+
+import { TemplateEditor } from '@folio/stripes-template-editor';
+import {
+  KeyValue,
+  Pane,
+  Paneset,
+  TextArea,
+} from '@folio/stripes/components';
+
+import StaffSlipForm from './StaffSlipForm';
+import TokensList from './TokensList';
+import tokens from './tokens';
+import {
+  CancelButton,
+  FooterPane,
+} from '../components';
+
+jest.mock('@folio/stripes/final-form', () => jest.fn(() => (component) => component));
+jest.mock('../components', () => ({
+  CancelButton: jest.fn(() => null),
+  FooterPane: jest.fn(() => null),
+}));
+Field.mockImplementation(jest.fn(() => null));
+Pane.mockImplementation(jest.fn(({ children, firstMenu, footer }) => (
+  <div>
+    {firstMenu}
+    {children}
+    {footer}
+  </div>
+)));
+
+describe('StaffSlipForm', () => {
+  const orderOfFieldExecution = {
+    description: 1,
+    template: 2,
+  };
+  const labelIds = {
+    name: 'ui-circulation.settings.staffSlips.name',
+    description: 'ui-circulation.settings.staffSlips.description',
+    display: 'ui-circulation.settings.staffSlips.display',
+    preview: 'ui-circulation.settings.staffSlips.form.previewLabel',
+    new: 'ui-circulation.settings.staffSlips.new',
+  };
+  const mockedHandleSubmit = jest.fn();
+  const mockedOnCancel = jest.fn();
+
+  afterEach(() => {
+    Field.mockClear();
```

Comment:
nit: maybe to clear mock for `Pane` as well? Not sure if it is necessary here but just in case.

Reply:
In this component we have only one `Pane`, and in all cases i check it with `toHaveBeenLastCalledWith` method, so it's not necessary in my opinion, but if you sleep better, I'll add it before merge =)

Reply:
it is okay, it is up to you